client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -294,6 +294,13 @@
                 scope.cacheNextItems = [];
                 scope.cachePreviousItems = [];
                 scope.limited = (monitoring.singleGroup || scope.group.type === 'highlights') ? false : true;
+                scope.stageLookup = {};
+
+                if (scope.group.type === 'stage') {
```

Comment:
Thanks for pointing out, it is supposed to be the incoming stage. I have double checked with PO's and they're happy that it will appear only on incoming and if that stage is not there in the desk monitoring view then they won't see the notifications.